docs/funcs/module.Call.md

```diff
@@ -31,6 +31,6 @@ Example:
 {{ module.Export "HelloWorld" }}
 
 // module-b
-{{ module.Call "module-b.HelloWorld" "Jared" }}
```

Comment:
hah oops, i thought that'd be obvious. good call

Reply:
def was not :D